bookwyrm/forms/links.py

```diff
@@ -37,10 +38,9 @@ def clean(self):
                     ),
                 )
         if (
-            not self.instance
-            and models.FileLink.objects.filter(
-                url=url, book=book, filetype=filetype
-            ).exists()
+            models.FileLink.objects.filter(url=url, book=book, filetype=filetype)
```

Comment:
Not for this PR, but I would propose dropping _filetype_ from this check. Thoughts?

(If file type is wrong and needs updating, the proposed change would ensure the pre-existing link is deleted first.)

Reply:
Agreed; I'll change that in a separate PR after this one.